Lib/dis.py

```diff
@@ -508,13 +503,9 @@ def _get_instructions_bytes(code, varname_from_oparg=None,
             elif deop in hascompare:
                 argval = cmp_op[arg>>4]
                 argrepr = argval
-            elif deop == FORMAT_VALUE:
-                argval, argrepr = FORMAT_VALUE_CONVERTERS[arg & 0x3]
-                argval = (argval, bool(arg & 0x4))
-                if argval[1]:
-                    if argrepr:
-                        argrepr += ', '
-                    argrepr += 'with format'
+            elif deop == CONVERT_VALUE:
+                argval = (None, str, repr, ascii)[arg]
+                argrepr = ('', 'str', 'repr', 'ascii')[arg]
```

Comment:
since oparg 0 isn't a thing (right?), maybe write this as:

```suggestion
                argval = (_, str, repr, ascii)[arg]
                argrepr = (_, 'str', 'repr', 'ascii')[arg]
```

Reply:
What is `_`? We might not use the 0th item, but it still needs to be defined.

Reply:
IIUC _ is the convention for ‘unused value’.

Reply:
For lvalues, sure, but this is an rvalue.